config/kibana.yml

```diff
@@ -65,6 +65,10 @@
 # headers, set this value to [] (an empty list).
 # elasticsearch.requestHeadersWhitelist: [ authorization ]
 
+# Header names and values that are sent to Elasticsearch. Any custom headers cannot be
+# overwritten by client-side headers, regardless of requestHeadersWhitelist configuration.
```

Comment:
Nit: change `requestHeadersWhitelist` to `elasticsearch.requestHeadersWhitelist` for consistency.